tools/docker-receptor.yml

```diff
@@ -29,32 +29,38 @@ services:
     volumes:
       - "../../docker-compose-cluster:/etc/receptor"
   receptor-1:
-    image: quay.io/ansible/awx-ee:devel
-    user: root
+    image: quay.io/awx/awx_devel:devel
+    user: "1000"
     container_name: tools_receptor_1
     hostname: receptor-1
     command: 'receptor --config /etc/receptor/awx-1-receptor.conf'
     links:
       - receptor-hop
     volumes:
       - "../../docker-compose-cluster:/etc/receptor"
+      - "/sys/fs/cgroup:/sys/fs/cgroup"
+    privileged: true
```

Comment:
@shanemcd Reusing your hacks to run podman inside of the container here.

I know that having the execution nodes on `quay.io/awx/awx_devel:devel` is non-ideal, and this is the next step we've planned:

https://github.com/ansible/awx/issues/10691

These execution nodes need a _subset_ of what's in the `quay.io/awx/awx_devel:devel` image. So we can get by without introducing a new image. However, I can see us refactoring this whole image ecosystem, and if we are doing that, we could start out doing things the right way.